crates/agent-sql/src/publications.rs

```diff
@@ -208,6 +208,123 @@ pub async fn resolve_spec_rows(
     .await
 }
 
+#[derive(Debug)]
+pub struct Tenant {
+    pub name: String,
+    pub captures_quota: i32,
+    pub derivations_quota: i32,
+    pub materializations_quota: i32,
+    pub collections_quota: i32,
+    pub captures_used: i32,
+    pub derivations_used: i32,
+    pub materializations_used: i32,
+    pub collections_used: i32,
+}
+
+#[derive(Debug)]
+struct TenantFromDB {
+    pub name: String,
+    pub captures_quota: i32,
+    pub derivations_quota: i32,
+    pub materializations_quota: i32,
+    pub collections_quota: i32,
+    // These are Option<i64> because even if we add `coalesce(.., 0) to the query
```

Comment:
nit: instead of doing this, you can tell sqlx in the query that these are non-null by using `foobar as "foobar!"`.

[here's one example](https://github.com/estuary/flow/blob/master/crates/agent-sql/src/publications.rs#L188-L193)

Then you can get rid of `TenantFromDB`

Reply:
Glorious, I tried searching for this but didn't find it

Reply:
Done